### DISCUSSION TITLE
Consistencia en el componente Feedback

### DISCUSSION THREAD

**Discussion by AnaMontes11** [Telefonica/mistica-design#1187] [Proposals]
### Goal

Valorar si tiene sentido tener unicamente dos componentes para Feedback:
1. `feedbackScreen`
2. `feedbackSection` <del>successFeedback </del>

y por paridad, cada uno de ellos con las siguientes variantes:

**feedbackScreen:**
type=
- Success
- Error
- Info

**feedbackSection:** 
type=
- Success
- Error
- Info

### Context

Actualmente para mostrar feedback tenemos 4 componentes:
1.SuccesFeedbackScreen
2.ErrorFeedbackScreen
3.InfoFeedbackScreen
4.SuccessFeedback

### Impact

Pensando en experiencia de diseño buscaría coherencia y simplicidad, tal y como propongo en el apartado Goal.

### Discussion Questions

A fullscreen quedan cubiertas todas las necesidades ya que tenemos 3 componentes para cada caso (`SuccesFeedbackScreen`, `ErrorFeedbackScreen`, `InfoFeedbackScreen`). ¿Tiene sentido 3 componentes por separado en vez de uno con estas 3 tipologías dentro?

Además de eso, para casos en los que no quieras/puedas usar una fullscreen, el componente `succesFeedback` está limitado unicamente a "success" lo que parece confuso.

**Comment by yceballost** [comment]
esto va en contra a cómo estamos ido modificando y separando los componentes de las variantes (inputs y proximamente botones), no?

Yo creo que tal y como está refleja bien lo que está en desarrollo, no haría cambios en ese sentido. Juntarlos supondría juntarlos también en desarrollo, no veo generar un breaking change en desarrollo por algo así.

En cuanto al successFeedback realmente tienes razón, solo existe para el success, igual debería existir para el resto de tipos? por otro lado, tampoco ha salido esa necesidad.

`feedbackSection` quizás me parece mejor nombre que el actual `successFeedback`

**Reply by AnaMontes11** [reply]
Con la actualización de los feedback ha vuelto a salir este tema. Voto a favor de renombrarlo como `feedbackSection`.

**Reply by AnaMontes11** [reply]
He creado las specs para los `successFeedback` (no las teniamos como tal) en una página separada para que quede más clara su diferencia [specs](https://www.figma.com/file/LpjgnU1xpmEzQvlXKwAHmn/%F0%9F%94%B8-Feedbacks-Specs?type=design&node-id=1939%3A16413&mode=design&t=TYUuIKUvvRcDigiv-1). Podríamos aprovechar para hacer renaming

**Comment by aweell** [comment]
Una propuesta adicional al renombre: Ahora mismo existe el componente `FeedbackScreen` que es una primitiva de los distintos `Feedback{TYPE}Screen`. Este `feedbackSection` podría jugar el mismo papel en vez de ceñirse únicamente al type `success`. Si luego se quisieran especializar los componentes, siguiendo la misma lógica podría ser `Feedback{TYPE}Section`.
